app/src/main/res/layout/item_color_attachment.xml

```diff
@@ -35,11 +36,24 @@
         app:layout_constraintTop_toTopOf="parent"
         tools:text="#5571 - User profile from SSO must not have password change option" />
 
+    <TextView
+        android:id="@+id/text_fields"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:visibility="gone"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="4dp"
+        app:layout_constraintTop_toBottomOf="@id/attachment_text"
```

Comment:
There are some questions related on using `@id` instead of `@+id`. In the end prefer to use `@+id` to don't care about it (widget disposition, etc.) - but looks like it doesn't have influence with the Constraint Layout.

Reply:
`@+id` should be use when "declaring" the id, for relations I prefer to use `@id` so it will fail if we remove the view with the declared id.